Tropos/Models/DailyForecast.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+@objc class DailyForecast: NSObject {
```

Comment:
Is it possible to mark this as `final`? Or is that disallowed by the `@objc` declaration?